config/locales/en.yml

```diff
@@ -79,6 +79,13 @@ en:
       errors:
         already_exists: "%{email} is already used by a different user."
         generic_error: Error occurred while updating the user's email. Please try again.
+    update_name:
+      title: Change name
+      given_name: Given name
```

Comment:
should this be now First name, as per the latest changes at https://github.com/alphagov/verify-self-service/pull/298 ... you then won't need to use the locales and can rely on the symbol names in the view